demo/index.html

```diff
@@ -48,8 +48,8 @@
           var combobox = document.querySelector('#demo2');
           combobox.items = elements;
 
-          combobox.addEventListener('value-changed', function() {
-            document.querySelector('#selected-value').innerHTML = combobox.value;
+          combobox.addEventListener('value-changed', function(e) {
+            document.querySelector('#selected-value').innerHTML = e.target.value;
```

Comment:
IMO, `combobox.value` is easier to get then `e.target.value`. I like that it explicitly tells that the value property is in the combobox.

Can we fix this without changing the example code this way?